### PR TITLE
Fix user lock feature on console

### PR DESCRIPTION
**Pull request by JayaShakthi97**
### Purpose
- Fix user lock feature functionality
- Fix lint warnings

### Related Issues
- https://github.com/wso2/product-is/issues/13379

### Checklist
- [ ] e2e cypress tests locally verified.
- [x] Manual test round performed and verified.
- [ ] UX/UI review done on final implementation.
- [ ] Documentation provided. (Add links if there's any)
- [ ] Unit tests provided. (Add links if there's any)
- [ ] Integration tests provided. (Add links if there's any)

### Related PRs
- None

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/features/users/components/user-profile.tsx [hunk 1]
```diff
@@ -607,43 +603,43 @@ export const UserProfile: FunctionComponent<UserProfilePropsInterface> = (
...
+                                "message")
+                    });
+
+                    return;
+                }
+
                 onAlertFired({
-                    description: error.response.data.description,
+                    description:
+                        editingAttribute?.name === "accountLocked"
```
Comment (brionmario) [inline apps/console/src/features/users/components/user-profile.tsx:629]: Use https://github.com/wso2/identity-apps/blob/69d11407f15f7c7a90861d04ee36f13e2fa946ae/modules/core/src/constants/profile-constants.ts#L72

**Review by brionmario — APPROVED** [review]

File: apps/console/src/features/users/components/user-profile.tsx (hunk 1 above)
Comment (JayaShakthi97) [reply apps/console/src/features/users/components/user-profile.tsx]: Fixed with da9e8dc029cfdf2d67de5b65f4a3faaa44c563a4

File: apps/console/src/features/users/components/user-profile.tsx [hunk 2]
```diff
@@ -255,21 +255,22 @@ export const UserProfile: FunctionComponent<UserProfilePropsInterface> = (
                     if (schemaNames[0] === "name") {
                         const name = schemaNames[1] && userInfo[schemaNames[0]] &&
                             userInfo[schemaNames[0]][schemaNames[1]] && (
-                                tempProfileInfo.set(schema.name, userInfo[schemaNames[0]][schemaNames[1]])
-                            );
+                            tempProfileInfo.set(schema.name, userInfo[schemaNames[0]][schemaNames[1]])
+                        );
```
Comment (mevan-karu) [inline apps/console/src/features/users/components/user-profile.tsx:259]: Check indentation and line breaks

Comment (JayaShakthi97) [reply apps/console/src/features/users/components/user-profile.tsx:259]: Fixed with a later commit fe1e21e443a8ffbab52c59243e01535fff12edcf

**Pull request merged**